source/upgrade-guide/upgrading-central-components.rst

```diff
@@ -176,6 +176,9 @@ When upgrading a multi-node Wazuh manager cluster, run the upgrade in every node
       If the ``/var/ossec/etc/ossec.conf`` configuration file was modified, it will not be replaced by the upgrade. You will therefore have to add the settings of the new capabilities manually. More information can be found in :doc:`/user-manual/index`.
 
 
+   .. warning::
```

Comment:
This note is too vague. Rather than a note we could could give the specifics on how to upgrade. The referred section is not enough clear on the steps to follow.

Reply:
Done!